xmrstak/misc/executor.cpp

```diff
@@ -600,7 +622,7 @@ void executor::ex_main()
 	if(jconf::inst()->GetVerboseLevel() >= 4)
 		push_timed_event(ex_event(EV_HASHRATE_LOOP), jconf::inst()->GetAutohashTime());
 
-	size_t cnt = 0;
+	size_t cnt = 0,i=0;
```

Comment:
please remove `i=0` it looks like it is not needed

Reply:
i is needed here because this variable is being used later on in one of the case statements in 2 places:

case EV_PERF_TICK:
                        for (i = 0; i < pvThreads->size(); i++)
                                telem->push_perf_value(i, pvThreads->at(i)->iHashCount.load(std::memory_order_relaxed),
                                pvThreads->at(i)->iTimestamp.load(std::memory_order_relaxed));

                        if((cnt++ & 0xF) == 0) //Every 16 ticks
                        {
                                double fHps = 0.0;
                                double fTelem;
                                bool normal = true;
                                for (i = 0; i < pvThreads->size(); i++)
                                {
                                        fTelem = telem->calc_telemetry_data(10000, i);
                                        if(std::isnormal(fTelem))
                                        {
                                                fHps += fTelem;
                                        }
                                        else
                                        {
                                                normal = false;
                                                break;
                                        }
                                }

                                if(normal && fHighestHps < fHps)
                                        fHighestHps = fHps;
                        }
                        break;